### ISSUE TITLE
Allow for output JSON to be written in `run` mode

### ISSUE THREAD

**Issue by geoffjentry** [broadinstitute/cromwell#1578]
Currently you can specify a file for the full workflow metadata to be written and while that does contain output that's pretty inconvenient for someone who is only interested in the output json. Allow for the output json to be sent to a file

**Comment by katevoss** [comment]
@geoffjentry The output JSON is printed to the CLI, right? Does that solve this problem or no?

**Comment by geoffjentry** [comment]
yes and no. I forget who asked for this, but this was a user request. They did *not* want it in the console output, and IMO that's a good idea.

**Comment by katevoss** [comment]
As a **workflow runner using the CLI**, I want **an option to send the output JSON into a separate file**, so that I can **avoid digging through the whole metadata for the information I need**.
- Effort: **?** @geoffjentry 
- Risk: **Small**
- Business value: **Small**

**Comment by geoffjentry** [comment]
hands on keyboard effort is relatively low. the real work is defining how to specify it, although now that we have the snazzy new CLI interface it's probably pretty simple to do so

**Comment by cjllanwarne** [comment]
@geoffjentry @kcibul @katevoss:

I'm removing the "WDL Developer Joy" label from this issue since I'm planning to use that for WDL team tickets and I believe this is a Cromwell change.

**Comment by magicDGS** [comment]
I am interested on this to import the oputput json and use it for analyze the data automatically after the workflow succeed.

In addition, related with this (if I should open a different issue, let me know), I found that the "output" entry in the metadata JSON (and in the console) it is pointing to the `cromwell-execution` folder instead of the custom `final_workflow_outputs_dir` specified in the command line. I expect that the folder structure in the JSON is in the provided one, to being able to operate with it instead of with the raw cromwell run.

**Comment by bolton-lab** [comment]
Yes if you are writing the ouputs json to the screen there should be an option to save this outputs json to file.  Only requires adding 1 extra command line options such as -j --json-outputs <file> and then you write to screen and to file.

We are talking about this outputs:
```
[2021-05-27 04:21:25,83] [info] SingleWorkflowRunnerActor workflow finished with status 'Succeeded'.
{
  "outputs": {
    "pindel.cwl.unfiltered_vcf": {
      "format": null,
      "location":
```
